package.json

```diff
@@ -87,7 +87,7 @@
     "dependencies": {
         "@types/react-helmet": "^5.0.8",
         "applicationinsights-js": "^1.0.20",
-        "axe-core": "3.1.1",
+        "axe-core": "3.2.1",
```

Comment:
I feel worried about the impact this might have on our persisted assessment data; have you already audited for whether there are any changes to the axe-generated bits of our persistent data that would need any special version migration considerations?

Reply:
Discussed offline: for the moment, we will keep as is since this change does not affect users. Moving forward, we may need to warn users or provide an additional UX if/when this happens.